.github/workflows/main.yml

```diff
@@ -350,6 +350,8 @@ jobs:
       - name: Run the Python tests
         run: make test-python
         timeout-minutes: 5
+      - name: Check for warnings in documentation
+        run: mkdocs build --strict
```

Comment:
This could also be replaced with `make build-docs`, let me know what you'd prefer.

Reply:
Let's use the make target.